web/src/main/java/se/inera/intyg/intygsadmin/web/controller/dto/CountStatusesForCareGiverRequestDTO.java

```diff
@@ -0,0 +1,18 @@
+package se.inera.intyg.intygsadmin.web.controller.dto;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.Builder;
+import lombok.Value;
+import se.inera.intyg.intygsadmin.web.service.status.NotificationStatusEnum;
+
+@Value
+@Builder
+public class CountStatusesForCareGiverRequestDTO {
```

Comment:
I think we need to update this with JsonPojoBuilder so that i can correctly deserialize?